site/docs/spark.md

```diff
@@ -38,12 +38,25 @@ Iceberg uses Spark's DataSourceV2 API for data source and catalog implementation
     Spark 2.4 can't create Iceberg tables with DDL, instead use the [Iceberg API](../api-quickstart).
 
 
-## Spark 2.4
+## Apache Spark 2.4
```

Comment:
Here's another case where "Apache" should be removed from a heading.

Reply:
Whoops, removed it.